metric_learn/itml.py

```diff
@@ -148,11 +148,23 @@ class ITML(_BaseITML, _PairsClassifierMixin):
   transformer_ : `numpy.ndarray`, shape=(num_dims, n_features)
       The linear transformation ``L`` deduced from the learned Mahalanobis
       metric (See function `transformer_from_metric`.)
+
+  threshold_ : `float`
+      If the distance metric between two points is lower than this threshold,
+      points will be classified as similar, otherwise they will be
+      classified as dissimilar.
+
+  classes_ : `list`
+      The possible labels of the pairs `LSML` can fit on. `classes_ = [-1, 1]`,
+      where -1 means points in a pair are dissimilar (negative label), and 1
+      means they are similar (positive label).
   """
 
   def fit(self, pairs, y, bounds=None):
     """Learn the ITML model.
 
+    The default threshold will be set to the mean of the bounds.
```

Comment:
to update, and make sure it says the same for all pairwise metric learners

Reply:
Do you mean we should use the same default threshold for ITML as for other pairwise metric learners ?  I agree there would be more coherence then
But then maybe we could still allow a method like `set_threshold_bounds` only for ITML that sets the threshold to the mean of the bounds ?